### DISCUSSION TITLE
Informasjonsmodell - bør denne vise "Organisasjon" som et objekt?

### DISCUSSION THREAD

**Discussion by OK-SverreM** [NorskHelsenett/Tillitsrammeverk#23] [General]
I informasjonsmodellen er en samling av sentral informasjon lagt inn i informasjonsobjektet "Behandlerrelasjon". Dette skjuler en del sentrale koblinger og relasjoner.
I datamodellen er "Organization" trukket ut som egen entitet. Bør det gjøres tilsvarende i informasjonsmodellen, slik at "Organisasjon" blir synlig som et eget informasjonsobjekt?

**Comment by steinarnoem** [comment]
Feature team ble enige om at temaet antagelig kommer opp i andre diskusjoner og at den derfor lukkes: 
Organisasjonen hvor pasienten mottar hjelp, eller organisasjonen hvor helsepersonellet jobber..
